Formula/folly.rb

```diff
@@ -40,25 +39,12 @@ def install
 
     cd "folly" do
       if MacOS.version == "10.11" && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
-        # Workaround for "no matching function for call to 'clock_gettime'"
-        # See upstream PR from 2 Oct 2016 facebook/folly#488
-        inreplace ["Benchmark.cpp", "Benchmark.h"] do |s|
-          s.gsub! "clock_gettime(CLOCK_REALTIME",
-                  "clock_gettime((clockid_t)CLOCK_REALTIME"
-          s.gsub! "clock_getres(CLOCK_REALTIME",
-                  "clock_getres((clockid_t)CLOCK_REALTIME", false
-        end
-
         # Fix "candidate function not viable: no known conversion from
         # 'folly::detail::Clock' to 'clockid_t' for 1st argument"
         # See upstream PR mentioned above
```

Comment:
`See upstream PR from 2 Oct 2016 facebook/folly#488` should be there instead of “mentioned above”